brownie/network/alert.py

```diff
@@ -90,7 +90,7 @@ def _loop(
                 start_value = value
                 if not repeat:
                     repeat = None
-                elif isinstance(repeat, int):
+                elif type(repeat) == int:
```

Comment:
It's more verbose, but I think preferable to use `isinstance(repeat, int) and not isinstance(repeat, bool)` - because [`Wei`](https://eth-brownie.readthedocs.io/en/stable/core-types.html#wei) is an `int` subclass and commonly used for return values, I can imagine a situation where the user is trying to set an alert with that type.

Reply:
Makes sense. Updated to reflect this.

I'll keep a lookout for possible improvements while we apply our usecase to it 👍